### ISSUE TITLE
Scan Results Page: Code Update [2]

### ISSUE THREAD

**Issue by ststimac** [webhintio/webhint.io#640]
Need to look into updating how the CSS is written for the scan results page, more specifically the overview panel on the right hand side. On smaller screens the way things are coded with calc and position: fixed and static, the menu is not accessible and you cannot close the menu via keyboard input.

* Mobile category menu needs to be updated anyway because it's not accessible via keyboard and there is no way to close or exit the menu.
* Melanie has some code from a separate project that will keep the functionality we have for the categories menu on a mobile screen the same + is accessible.

Make mobile category menu fully accessible (moved to #661)
- [x] Go through scan results page CSS. I noticed some unnecessary styling being applied in places. 
- [x] Edit "Expand/Close Button" height. Seeing a cropping issue in Chrome, Edge. FF: 
![screen shot 2019-02-21 at 3 39 37 pm](https://user-images.githubusercontent.com/18073131/53209373-1bb7d880-35ef-11e9-84d1-f6a525f1d9b3.png)

**Comment by ststimac** [comment]
Just making some notes here after investigating this further. For the mobile category menu, the accessibility issue we have is that when you've opened the menu via keyboard and navigate to one of the anchor links, when you hit enter, the category menu doesn't disappear and focus moves to the page underneath it, to the category you navigated to, and so you can't escape the category menu to get to the results content.

**Comment by ststimac** [comment]
This is going to take two PRs split across the hint and webhintio repo.

**Comment by ststimac** [comment]
@sarvaje I just met with Melanie to talk about what's happening on the mobile view for the scan results page. 

When a user has the categories expanded like this: 
<img width="512" alt="screen shot 2019-03-06 at 11 22 41 am" src="https://user-images.githubusercontent.com/18073131/53907563-50219080-4002-11e9-8066-2dacb837814e.png">

And is using the keyboard to tab through the the categories, if they hit "enter" to tab into a section, the categories menu is not closing when enter is being pressed. This is only happening via the keyboard, if someone is using a mouse to click the categories, a click event is happening. So we need an event for focusing and hitting enter that then closes the category menu. 

Let me know if that makes sense and @molant let me know if I should open a separate issue for next sprint that focuses on the JavaScript.

**Comment by sarvaje** [comment]
@ststimac it makes sense :)

**Comment by molant** [comment]
> let me know if I should open a separate issue for next sprint that focuses on the JavaScript.

If you have more tasks than this one then open a new task so we can track it accordingly. If it is just this one then don't think it's worth it.

Thanks!

**Comment by ststimac** [comment]
@molant I have finished the other two tasks that were a part of this overall issue so I will create a new one.
